tests/validation/tests/v3_api/test_zero_downtime.py

```diff
@@ -0,0 +1,352 @@
+from .common import *
+import pytest
+import math
+import time
+from .test_rke_cluster_provisioning import create_custom_host_from_nodes
+
+HOST_NAME = os.environ.get('RANCHER_HOST_NAME', "testZeroDT")
+DRAIN = os.environ.get('RANCHER_DRAIN_INPUT', "False")
+MAX_UNAVAILABLE_WORKER = os.environ.get("RANCHER_MAX_UNAVAILABLE_WORKER", "10%")
+MAX_UNAVAILABLE_CONTROLPLANE = os.environ.get("RANCHER_MAX_UNAVAILABLE_CONTROLPLANE", "1")
+NODE_COUNT_CLUSTER = os.environ.get("RANCHER_NODE_COUNT_CLUSTER", 3)
+host = "test" + str(random_int(10000, 99999)) + ".com"
+path = "/name.html"
+namespace = {"p_client": None, "ns": None, "cluster": None, "project": None,
+             "nodes": []}
+backup_info = {"backupname": None, "backup_id": None, "workload": None,
+               "backupfilename": None, "etcdbackupdata": None}
+K8S_VERSION_PREUPGRADE = os.environ.get("RANCHER_K8S_VERSION_PREUPGRADE", "v1.15.12-rancher2-2")
```

Comment:
nit: move this up so all the env vars are first, followed by other lowercase vars